src/Symfony/Component/Mime/Tests/Part/TextPartTest.php

```diff
@@ -87,6 +88,35 @@ public function testEncoding()
         ), $p->getPreparedHeaders());
     }
 
+    public function testCustomEncoding()
+    {
+        $p = new TextPart('content', 'utf-8', 'plain', 'test_encoding');
+        $p->addEncoder('test_encoding', new class() implements ContentEncoderInterface {
+            public function encodeByteStream($stream, int $maxLineLength = 0): iterable
+            {
+                while (!feof($stream)) {
+                    yield fread($stream, 16372);
+                }
+            }
```

Comment:
Does the test actually cover the call to this method?

Reply:
Yes, this line covers that:
```
 $this->assertEquals('CONTENT', implode('', iterator_to_array($p->bodyToIterable())));
```